website/docs/docs/deploy/continuous-integration.md

```diff
@@ -0,0 +1,60 @@
+
+Using Slim CI helps:
+
+- Provide increased confidence and assurances that project changes will work as expected in production.
+- Reduce the time it takes to push code changes to production, through build and test automation, leading to better business outcomes.
+- Allow organizations to make code changes in a standardized and governed way that ensure code quality without sacrificing speed.
+
+## How Slim CI works
+
+When you [set up Slim CI jobs](/docs/deploy/slim-ci-jobs#set-up-slim-ci-jobs), dbt Cloud listens for webhooks from your Git provider indicating that a new PR has been opened or updated with new commits. When dbt Cloud receives one of these webhooks, it enqueues a new run of the Slim CI job. If you want CI checks to run on each new commit, you need to mark your PR as **Ready for review** &mdash; draft PRs _don't_ trigger CI jobs. 
+
+dbt Cloud builds and tests the models affected by the code change in a temporary schema using the prefix `dbt_cloud_pr_`. This process ensures that the code builds without error and that it matches the expectations as defined by the dbt tests. The unique schema name follows the naming convention `dbt_cloud_pr_<job_id>_<pr_id>` (for example, `dbt_cloud_pr_1862_1704`) and can be found in the run details for the given run, as shown in the following image:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/using_ci_dbt_cloud.png" width="90%"title="Viewing the temporary schema name for a run triggered by a PR"/>
+
+When the Slim CI run completes, you can view the run status directly from within the pull request. dbt Cloud updates the pull request in GitHub, GitLab, or Azure DevOps with a status message indicating the results of the run. The status message states whether the models and tests ran successfully or not. 
+
+dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has database or schema customization using the [generate_database_name](/docs/build/custom-databases#generate_database_name) or [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macros, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Temp PR schema limitations](#temp-pr-schema-limitations).
+
+## Differences between Slim CI jobs and other deployment jobs
+
+The [dbt Cloud scheduler](/docs/deploy/job-scheduler) executes Slim CI jobs differently from other deployment jobs in these important ways:
+
+- **Concurrent CI checks** &mdash; Slim CI runs triggered by the same dbt Cloud Slim CI job execute concurrently (in parrallel), when appropriate
+- **Smart cancellation of stale builds** &mdash; automatically cancels stale, in-flight Slim CI runs when there are new commits to the PR
+- **Run slot treatment** &mdash; Slim CI runs don't consume a run slot
```

Comment:
with the right sidebar (page toc with links to those same sections) and this being such a short section with these subsections right below it, i don't think we need links here. 

since all list items consistently don't use punctuation, it's stylistically ok to leave them out